### PR TITLE
Bug fix/implement windows maintenance tests

### PR DESCRIPTION
**Pull request by dothebart**
we now load the JSON data on windows via RC files. We have a shellscript converting the header data into the RC file.

### CODE REVIEW

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/5791/

File: tests/Maintenance/json.rc [hunk 1]
```diff
@@ -0,0 +1,10877 @@
+#define IDS_CURRENT 1
+#define IDS_DBSERVER0001 2
+#define IDS_DBSERVER0002 3
+#define IDS_DBSERVER0003 4
+#define IDS_PLAN 5
+#define IDS_SUPERVISION 6
```
Comment (Dronplane) [inline tests/Maintenance/json.rc]: All these defines  could be replaced by 
 #include "jsonresource.h"

File: tests/Maintenance/MaintenanceTest.cpp [hunk 2]
```diff
@@ -325,6 +365,7 @@ TEST_F(MaintenanceTestActionDescription, retrieve_nonassigned_key_from_actiondes
     auto bogus = desc.get("bogus");
     ASSERT_TRUE(bogus == "bogus");
   } catch (std::out_of_range const& e) {
+    e;
```
Comment (Dronplane) [inline tests/Maintenance/MaintenanceTest.cpp]: exception variable name could be just omitted if we are not interested in exception details

File: tests/Maintenance/MaintenanceTest.cpp [hunk 3]
```diff
@@ -340,6 +381,7 @@ TEST_F(MaintenanceTestActionDescription, retrieve_nonassigned_key_from_actiondes
...
```
Comment (Dronplane) [inline tests/Maintenance/MaintenanceTest.cpp]: exception variable name could be just omitted if we are not interested in exception details

File: tests/Maintenance/MaintenanceTest.cpp [hunk 4]
```diff
@@ -68,6 +68,45 @@ char const* dbs2Str =
 #include "DBServer0003.json"
     ;
 
+int loadResources(void) {return 0;}
+
+#else // _WIN32
+#include <unicode/locid.h>
+#include <unicode/uchar.h>
+#include <unicode/unistr.h>
```
Comment (Dronplane) [inline tests/Maintenance/MaintenanceTest.cpp]: Do we still need this includes from unicode here?

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/5793/

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/5794/

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/5799/

**Comment by dothebart** [comment]
forgot to push - http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr-mac-onlycatch/3831/ green now.

**Pull request merged**